fs/path.py

```diff
@@ -256,6 +257,29 @@ def combine(path1, path2):
     return "{}/{}".format(path1.rstrip('/'), path2.lstrip('/'))
 
 
+def parts(path):
```

Comment:
Equivalent to https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.parts